### ISSUE TITLE
ignore spawn chance for spawn eggs

### ISSUE THREAD

**Issue by thimblebird** [nvb-uy/YSNS#10]
spawn eggs are included in set spawn chance

**Comment by nvb-uy** [comment]
Yes, it's just global spawns, so there isn't a way currently for me to add separate spawn rules. I'll try to do this in the future

**Comment by thimblebird** [comment]
Ah okay. Shouldn't the issue stay open then?
